website/src/content/docs/guides/how-biome-works.mdx

```diff
@@ -130,10 +133,12 @@ For example, you have a project that contains two directories called `backend/`
 }
 ```
 
-When you run the script `format` inside `frontend/package.json`, the working directory resolved by that script will be `frontend/`, the globs `src/**/*.js` and `src/**/*.ts` will have as "base" directory `frontend/`.
+When you run the script `format` inside `frontend/package.json`,
+the working directory resolved by that script will be `frontend/`,
+the globs `src/**/*.js` and `src/**/*.ts` will have as "base" directory `frontend/`.
 
 :::note
-When both `ignore` and `include` are specified, Biome gives **precedence** to `ignore`.
+When both `include` and `ignore` are specified, Biome ignores a file if teh file doesn't match a glob in `include` or does match a glob in `ignore`.
```

Comment:
I preferred the previous phrase, where we highlight the fact that we give precedence to `ignore`. 

That's because if a file matched a glob both in `ignore` and `include`, `ignore` has precedence and wins